lib/src/touch_spin.dart

```diff
@@ -30,8 +30,8 @@ class TouchSpin extends StatefulWidget {
     this.addIcon = const Icon(Icons.add),
     this.iconPadding = const EdgeInsets.all(4.0),
     this.textStyle = const TextStyle(fontSize: 24),
-    this.iconActiveColor,
-    this.iconDisabledColor,
+    this.iconActiveColor = Colors.green,
+    this.iconDisabledColor = Colors.grey,
```

Comment:
Colors should be driven by the theme, not hard-coded.
Was: `Theme.of(context).disabledColor`